src/main/java/org/aksw/gerbil/http/HttpManagement.java

```diff
@@ -93,14 +114,9 @@ public synchronized static HttpManagement getInstance() {
         return instance;
     }
 
-    protected InterruptingObserver interruptingObserver;
-    protected CloseableHttpClient client;
-    protected String userAgent;
-    protected Map<String, Long> blockingDomainTimestampMapping = new HashMap<>();
-
     protected HttpManagement(InterruptingObserver interruptingObserver, String userAgent) {
         this.interruptingObserver = interruptingObserver;
-        this.client = generateHttpClientBuilder().build();
+        this.client = generateHttpClientBuilder().setUserAgent(userAgent).build();

```

Comment:
Since the `userAgent` is set here now, [Line 195](https://github.com/dice-group/gerbil/blob/4336aaeb1dfc643fdbd8ce8d737b838444bd5659/src/main/java/org/aksw/gerbil/http/HttpManagement.java#L195) can removed. Since it is never initialized, it should be null there anyway.

Reply:
Actually, the variable could be removed then.

Reply:
Good point.

I missed to check whether the `userAgent` attribute is necessary or not and it turned out, that it is necessary since the `generateHttpClientBuilder` method is called at
https://github.com/dice-group/gerbil/blob/master/src/main/java/org/aksw/gerbil/annotator/impl/tagme/TagMeAnnotator.java#L99
So I had to roll back the changes of this part. Thanks for the hint.